DotnetFoundation/DotnetFoundation.Api/Controllers/AuthenticationController.cs

```diff
@@ -8,35 +8,35 @@ namespace DotnetFoundation.Api.Controllers;
 [Route("/api/auth")]
 public class AuthenticationController : ControllerBase
 {
-  private readonly IAuthenticationService _authenticationService;
-  public AuthenticationController(IAuthenticationService authenticationService)
-  {
-    _authenticationService = authenticationService;
-  }
+    private readonly IAuthenticationService _authenticationService;
+    public AuthenticationController(IAuthenticationService authenticationService)
+    {
+        _authenticationService = authenticationService;
+    }
```

Comment:
I don't think this is the standards we follow. The tab is 4 spaces in length.

Reply:
It already had some space so updated it add rest so warning is fixed
![image](https://github.com/OsmosysSoftware/dotnet-foundation/assets/121787291/f908324d-c6c8-4b16-86ae-5e295ac35beb)